src/Service.Tests/DatabaseSchema-MySql.sql

```diff
@@ -152,7 +152,8 @@ CREATE TABLE trees (
 
 CREATE TABLE fungi (
     speciesid int PRIMARY KEY,
-    region text
+    region text,
+    habitat varchar(6)
```

Comment:
are we encountering this perf issue on pg/mysql?

Reply:
nope. this is specific to MsSql